vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -1387,6 +1398,12 @@ void CompositionViewComponentView::updateProps(
     m_visual.Opacity(newViewProps.opacity);
   }
 
+  if (oldViewProps.accessibilityValue.text.has_value() &&
```

Comment:
> Separately, whenever the component view updates its value it needs to notify the automation peer that its value has been changed.

Does this change addresses this issue - whenever updateProps() is called, if accessibilityValue.text has changed, it will be modified and UpdateUiaProperty() will be called.